src/graphql/types/scalar/currency-unit.ts

```diff
@@ -0,0 +1,11 @@
+import { GT } from "@graphql/index"
+
+const CurrencyUnit = GT.Enum({
```

Comment:
why is this added if its also deprecated instantly?

Reply:
https://github.com/GaloyMoney/galoy/pull/2414#discussion_r1135669342